Specs/Renderer/TextureSpec.js

```diff
@@ -477,21 +481,21 @@ describe(
         context: context,
         fragmentShader: fs,
         uniformMap: uniformMap,
-      }).contextToRender([0, 255, 0, 255]);
+      }).contextToRender([255, 255, 255, 255]);
```

Comment:
White is not the best color for testing because it can lead to false positives. Can this test use the green image instead and can `White64x64_Basis255` be removed?

Reply:
Updated